src/Misc/layoutbin/en-US/strings.json

```diff
@@ -262,6 +262,9 @@
   "DeploymentPoolNotFound": "Deployment pool not found: '{0}'",
   "DeprecatedNode6": "This task uses Node 6 execution handler, which will be removed March 31st 2022. If you are the developer of the task - please consider the migration guideline to Node 10 handler - https://aka.ms/migrateTaskNode10 (check this page also if you would like to disable Node 6 deprecation warnings). If you are the user - feel free to reach out to the owners of this task to proceed on migration.",
   "DeprecatedRunner": "Task '{0}' is dependent on a task runner that is end-of-life and will be removed in the future. Authors should review Node upgrade guidance: https://aka.ms/node-runner-guidance.",
+  "DeprecationMessage": "Task '{0}' version {1} ({2}@{1}) is deprecated.",
+  "DeprecationMessageHelpUrl": "Please see {0} for more information about this task.",
+  "DeprecationMessageRemovalDate": "This task will be removed. From {0}, onwards it may no longer be available.",
```

Comment:
Suggestion: I don't think we need "This task will be removed".  

I'm thinking: 
"Task X version Y is deprecated.  From {0}, onwards it may no longer be available" 

is sufficient.

@geekzter

Reply:
According to the warning message, it should be clear that the task will definitely be removed after the specified date, but the term of the removal, which starts from the specified date, is not defined.